content/en/contribute/code/workflow.md

```diff
@@ -69,6 +69,22 @@ All features and bug fixes must have at least one unit test. All features must h
 
 Those are minimums. Our ultimate goal is to have fully-automated release testing, allowing for fast, confident delivery of completed code. If your work would be included in a release/regression test, create an e2e test for it. QA engineers will use that as a template of sorts to create additional automated tests at their discretion.
 
+### Acceptance Testing
+
+  1 . Assign it to yourself.
+  2. Install the PR branch to test against
+  3. If the issue fails AT then notify the original developer and move the issue back to "In progress".
+  4. Once the issue passes AT notify the original developer and move the issue to "Ready to merge"
+  5. The developer will then "Squash and Merge" the PR, delete the PR branch and, if a backport is required cherry-pick the merged commit back to the release branches it's required in.
+  6. Check if there is an e2e test for the feature, if aplicable.
+  7. Close the issue, which will move it to `Done`
```

Comment:
I think this duplicates a lot of the information in the "AT in progress" section below. Could you have a look at merging the two? I think it fits better down below but there may be additional useful information in this write up.

Reply:
Agree- complete duplicate. Didn't notice that section - will remove this.